lib/pf/Switch/Mikrotik.pm

```diff
@@ -257,8 +260,8 @@ Overloading L<pf::Switch>'s implementation because Mikrotik have his own radius
 
 Don't forget to fill /usr/share/freeradius/dictionary.mikrotik with the following attributes:
 
-ATTRIBUTE       Mikrotik-Wireless-VlanID                26      integer
-ATTRIBUTE       Mikrotik-Wireless-VlanIDType            27      integer
+ATTRIBUTE       Mikrotik-Wireless-VLANID                26      integer
+ATTRIBUTE       Mikrotik-Wireless-VLANID-Type           27      integer
```

Comment:
It's already the case in the default freeradius mikrotik dictionary:
ATTRIBUTE       Mikrotik-Wireless-VLANID                26      integer
ATTRIBUTE       Mikrotik-Wireless-VLANIDtype            27      integer
ATTRIBUTE       Mikrotik-Wireless-VLANID-Type           27      intege

Reply:
I understand that PF provided the MikroTik vendor attributes ahead of them being included in FreeRADIUS. This lead to a patch a while ago where the PF attributes were changed to reference the slightly different format.

All references to the 'Mikrotik-Wireless-VlanIDType' attribute were changed to 'Mikrotik-Wireless-VLANID-Type' (includes a hyphen), with exception to the comment in the MikroTik module.

I was actually also under the impression that attribute names were case sensitive, but this seems to be incorrect. I would however still like to recommend that we then either completely remove the comment about having to possibly define the attributes, or (as I'm recommending) renaming the attribute name in the comment to match all other changes.

```
[admin@packetfence2 ~]# grep -i Mikrotik-Wireless-VlanID-*Type Mikrotik.pm.orig
ATTRIBUTE       Mikrotik-Wireless-VlanIDType            27      integer
            'Mikrotik-Wireless-VLANID-Type' => "0",
```

Reply:
PS: I'll work on changes to the documentation (docs/network/networkdevice/mikrotik.asciidoc) and try to update the patch to include these.

Reply:
> I understand that PF provided the MikroTik vendor attributes ahead of them being included in FreeRADIUS. This lead to a patch a while ago where the PF attributes were changed to reference the slightly different format.
> 
> All references to the 'Mikrotik-Wireless-VlanIDType' attribute were changed to 'Mikrotik-Wireless-VLANID-Type' (includes a hyphen), with exception of the comment in the MikroTik module.
> 
> I was actually also under the impression that attribute names were case sensitive, but this seems to be incorrect. I would however still like to recommend that we then either completely remove the comment about having to possibly define the attributes, or (as I'm recommending) renaming the attribute name in the comment to match all other changes.
> 
> ```
> [admin@packetfence2 ~]# grep -i Mikrotik-Wireless-VlanID-*Type Mikrotik.pm.orig
> ATTRIBUTE       Mikrotik-Wireless-VlanIDType            27      integer
>             'Mikrotik-Wireless-VLANID-Type' => "0",
> ```
> 
> PS: FreeRADIUS 3.0.21 as part of a Debian 9 installation following the PF installation instructions does not include the attribute name without the '-':
> 
> ```
> [admin@packetfence2 ~]# grep -Ri Mikrotik-Wireless-VlanID-*Type /usr/share/freeradius
> /usr/share/freeradius/dictionary.mikrotik:ATTRIBUTE     Mikrotik-Wireless-VLANID-Type           27      integer
> ```

ok, let's keep like that for now